doc/ch-run.rst

```diff
@@ -92,6 +93,9 @@ setuid or setcap helpers, even for mounting SquashFS images with FUSE.
     bind-mounted into it. If this is specified, no such temporary files are
     created and the image’s files are exposed.
 
+  :code:`-s`, :code:`--storage DIR`
+    Set the storage directory (see section 6.5 for important details).
```

Comment:
The man pages don’t have section numbers; also this refers to a section in a different man page.